packages/geo/src/props.js

```diff
@@ -118,6 +119,7 @@ const commonChoroplethDefaultProps = {
     label: 'id',
     value: 'value',
     colors: 'PuBuGn',
+    domain: [0, 1000000],
```

Comment:
This is quite an arbitrary domain, I think we can remove this and let users add their own,  otherwise I'm afraid that people won't understand where does the steps come from.